temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
**Suggestion:** Handle potential exceptions that can be raised during file operations, such as 'FileNotFoundError' or 'PermissionError', to make the code more robust. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
import json

def reset_files(filepath, file_type="txt"):
    try:
        with open(filepath, "w") as output_file:
            if file_type == "txt":
                output_file.write("")
            elif file_type == "json":
                output_file.write(json.dumps({}))
            else:
                print("ERROR: Invalid file type")  # or raise ValueError("ERROR: Invalid file type")
    except (FileNotFoundError, PermissionError) as e:
        print(f"Failed to reset file: {e}")
```